src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,93 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is better to divide all the logic into three methods:
1. Reading information from a file, in what form it is, and in this way, it can be read into a string, it can be into an array of strings.
2. Information processing and report creation.
3. Writing to the file of the created report.